x/nft/spec/01_concepts.md

```diff
@@ -0,0 +1,13 @@
+<!--
+order: 1
+-->
+
+# Concepts
+
+## Class
+
+`x/nft` module defines a struct `Class` to describe the common characteristics of a class of nft, under this class, you can create a variety of nft, which is equivalent to an erc721 contract for Ethereum. The design is defined in the [ADR 043](../../../docs/architecture/adr-043-nft-module.md).
```

Comment:
```suggestion
`x/nft` module defines a struct `Class` to describe the common characteristics of a class of nft, under this class, you can create a variety of nft, which is equivalent to an erc721 contract for Ethereum. The design is defined in the [ADR 043](https://github.com/cosmos/cosmos-sdk/blob/master/docs/architecture/adr-043-nft-module.md).
```

This cannot be a relative link; otherwise it is broken (e.g. https://docs.cosmos.network/master/modules/authz the `ADR 30` does not work in the authz doc because it was relative)